config/routes.rb

```diff
@@ -1,9 +1,20 @@
 Rails.application.routes.draw do
-  get 'users/index'
-  get 'users/new'
+  # get 'users/index'
+  # get 'users/new'
   devise_for :users
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-  # root to: "users#index"
-  # Defines the root path route ("/")
-  # root "articles#index"
+  unauthenticated :user do
+    root to: "users#index"
+  end
+
+  authenticated :user do
+    root to: "categories#index", as: :authenticated_root
+  end
+  resources :categories do
+    resources :expenditures
+  end
```

Comment:
- [ ] Please add only necessary routes. You can do it by using `only`
For example, in categories, you are not using the edit controller but it is showing when ı run rails routes.